tests/unit/advanced/test_metadata.py

```diff
@@ -136,3 +137,43 @@ def test_edge_multiple_partition_and_clustering_keys(self):
             'FROM from_label((pk1, pk2), c1, c2) ',
             tm.as_cql_query()
         )
+
+    def test_metadata_query_metadata_timeout(self):
```

Comment:
why adding test inside `GraphMetadataToCQLTests` ? 

we are not running those tests, AFAIK

Reply:
Thanks, moved out to a separate class.